python/delta/tables.py

```diff
@@ -127,12 +204,21 @@ def vacuum(self, retentionHours=None):
         else:
             return DataFrame(jdt.vacuum(retentionHours), self._spark._wrapped)
 
+    @since(0.4)
     def history(self, limit=None):
         """
         Get the information of the latest `limit` commits on this table as a Spark DataFrame.
         The information is in reverse chronological order.
-        :param limit:
-        :return: DataFrame of the history
+
+        Example::
+
+            fullHistoryDF = deltaTable.history()    # get the full history of the table
+
+            lastOperationDF = deltaTable.history(1) # get the last operation
+
+        :param limit: Optional, number of latest commits to returns in the history.
+        :return: table's commit history
```

Comment:
Maybe some more information on what to do with this or parse this, or where they can find what the format of this is.

Reply:
done.